app/helpers/documents_helper.rb

```diff
@@ -32,19 +32,19 @@ def attached_file_formatter(document)
     case document.state
     when 'Completed'
       content_tag(:a, class: 'attached_file', id: "file_#{document.id}", href: document_path(document), target: :blank, title: 'Download File', 'data-id' => document.id) do
-        content_tag(:span, '', class: 'glyphicon glyphicon-file')
+        content_tag(:span, '', class: 'fa fa-file')
       end
     when 'Processing'
-      content_tag(:span, '', class: 'glyphicon glyphicon-refresh spin', style: 'cursor:auto')
+      content_tag(:span, '', class: 'fas fa-cog fa-spin processing', style: 'cursor:auto')
     else
-      content_tag(:span, '', class: 'glyphicon glyphicon-alert danger', style: 'cursor:auto')
+      content_tag(:span, '', class: 'fas fa-exclamation-triangle', style: 'cursor:auto;color:red;')
     end
   end
 
   def edit_formatter(document)
```

Comment:
Can you change the edit and delete formatters to use stylized buttons in a single "Actions" column for consistency? Also can you change them to `link_to`s with the corresponding path helper so that we're not using JavaScript in `documents.js.coffee` to send the request?

Reply:
Edit and delete have been combined into an actions column. I'm going to put in a story in the next version for javascript changes like this because I think it's out of scope for the bootstrap upgrade.